src/libraries/System.Transactions.Local/src/System/Transactions/Configuration/DefaultSettingsSection.cs

```diff
@@ -10,9 +10,9 @@ internal sealed class DefaultSettingsSection // ConfigurationSection
 
         internal static DefaultSettingsSection GetSection() => s_section;
 
-        public string DistributedTransactionManagerName { get; set; } = ConfigurationStrings.DefaultDistributedTransactionManagerName;
+        public static string DistributedTransactionManagerName { get; set; } = ConfigurationStrings.DefaultDistributedTransactionManagerName;
```

Comment:
@roji should weigh in on whether any of these changes are going to conflict with his work.